app/models/nested_facet.rb

```diff
@@ -6,7 +6,21 @@ def initialize(facet, values)
     super
   end
 
+  def facet_options
+    allowed_values.inject([{ text: "All #{pluralized_facet_short_name}", value: "" }]) do |options, allowed_value|
```

Comment:
I'd suggest pulling this out into a well named variable, e.g.

```suggestion
    all_option = { text: "All #{pluralized_facet_short_name}", value: "" }
    allowed_values.inject([all_option]) do |options, allowed_value|
```